bluebrain/deployment/environments/externals.yaml

```diff
@@ -6,6 +6,7 @@ spack:
         include:
           - arm-forge
           - apptainer
+          - binutils
```

Comment:
IIRC, didn't we have that module in `compilers`, as `gcc` really should depend on the `binutils` module?

Reply:
Looks like yes: c355bd6792739d5d9f1cd8910ed80de21e51e243